src/controllers/auth.controller.js

```diff
@@ -0,0 +1,66 @@
+const authService = require('../services/auth.service');
+
+const register = async (req, res) => {
+  const userBody = req.body;
+  const newUser = await authService.register(userBody);
```

Comment:
Consider adding error handling for the 'register' function. If 'authService.register' fails, it could result in an unhandled promise rejection.